src/main/python/rlbot/utils/class_importer.py

```diff
@@ -79,18 +78,11 @@ def load_external_module(python_file):
 
     dir_name = os.path.dirname(python_file)
     module_name = os.path.splitext(os.path.basename(python_file))[0]
-    keys_before = set(sys.modules.keys())
```

Comment:
I remember using things from this file in RLBotTraining. Have you tested whether there is an impact?
https://github.com/RLBot/RLBotTraining/blob/580e162e54f41d278319bf4f68d99dab7f3b7ed8/rlbottraining/exercise_runner.py#L136

Reply:
I ran a match with a bunch of bots from the bot pack, but did not test anything in RLBotTraining.  Thanks for calling this out, I'll test training.

Reply:
Training does still work when executing `rlbottraining run_module .\hello_world_training.py`, which I believe exercises the code in question.